### ISSUE TITLE
Map Pull Requests

### ISSUE THREAD

**Issue by CharlieAtlas** [triplea-game/triplea#8245]
I am guessing anyone that would actually do it is more knowledgeable than me, so you can make your own judgment!  But I thought it would be worth compiling old map PRs since it is likely that no one has looked at some of them for a few months (or years!)

These all look like they can be merged with no issues.   

- [ ] https://github.com/triplea-maps/battle_for_arda/pull/3            
- [x] https://github.com/triplea-maps/new_world_order_lebowski_edition/pull/4  
- [ ] https://github.com/triplea-maps/world_war_ii_v5_1942/pull/7
- [x] https://github.com/triplea-maps/sleeping_giant/pull/3
- [ ] https://github.com/triplea-maps/stellar_forces/pull/3
- [ ] https://github.com/triplea-maps/another_world/pull/2
- [x] https://github.com/triplea-maps/ultimate_world/pull/1

This one should probably get a YAML update, but other than that it should be able to be merged.     
- [ ] https://github.com/triplea-maps/steampunk_advanced/pull/1 

I am less sure on this one, but feel like it can probably be merged. 
- [x] https://github.com/triplea-maps/european_war/pull/2

Created by me, so i feel like someone else should look at them before merging them.  
- [ ] https://github.com/triplea-maps/domination_1914_no_mans_land/pull/10
- [ ] https://github.com/triplea-maps/world_war_ii_global/pull/42

Beyond my knowledge
- [ ] https://github.com/triplea-maps/1914-cow-empires/pull/1

edit1: added formatting to keep track of what is done
edit2: adding new ones as well periodically

**Comment by DanVanAtta** [comment]
Those PRs look to have slipped through the cracks.

CharlieAtlas, I sent you an invite to 'MapAdmins' that will give you write access to all maps. You can then merge those PRs, just be careful about it and try to test/ensure the changes are good. Assuming you're pretty confident in those PRs, you can merge them in.

**Comment by CharlieAtlas** [comment]
I don't know if i want that much power!!!  Primarily, I am not sure i would be able to properly test them.  I already had issues with that in one PR of my own.  

Is there any way to know the original map maker, to know if a PR is them?  There were a couple that looked like they could have been deliberately the way they were before the PR.  Does that even matter at this point?

**Comment by DanVanAtta** [comment]
Some configs can be tested by checking that the map simply launches. Others are going to be more involved and will need to go into specific game mechanics.

Getting things into a good & working state is more important than original intent, a broken map is not valuable. Most maps are just uploaded but the original author has wondered off. The reality is most maps are only maintained by the map admin team. If you can find of the original map authors - that is really great

**Comment by beelee1** [comment]
@CharlieAtlas I think european war is navalland or schultz at the triplea site. Iirc there were two that were the same and we had Dan 86 one of them.

**Comment by CharlieAtlas** [comment]
Looks like European War is gone, so I checked it off.  I merged and checked off the 3 i was super confident of.  Stellar Forces and Another World are territory effect images.  Is there anything I can do to test them?

I am not sure the WW2v5 one should actually be done, as I commented in the issue associated with it.  

I don't seem to have permission for Steam Punk or Battle for Arda.  The should both be fine though.
